src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java

```diff
@@ -721,6 +788,21 @@ public void login(String url, String userName, char[] password, char[] token) th
         }
     }
 
+    @Override
+    public boolean isAuthorized() {
+        try {
+            client.secrets().list();
+            return true;
```

Comment:
I prefer the current impl: the natural order is an operation after the other unless you have an exception which lets the execution to continue in the catch. The same goes for the return after the catch: imho that's unnatural. We have either success or not and therefore either return true in the natural order of operations or return false in case there's an exception. A return outside the catch feels & looks like code that's not related.